services/database/user.ts

```diff
@@ -35,4 +42,177 @@ async function getUsers() {
     return users;
 }
 
-export { getUser, getUsers };
+/* 
+TODO (6/30/21): test passing userId in newTeacherData instead of through connect key
+*/
+
+async function updateUser(user: UserInput): Promise<User> {
+    /*
+    Flow:
```

Comment:
How are you guys ensuring mutual exclusion (preventing an user that's already a teacher from being able to create another role that's not a teacher)?

Reply:
added a role field to the user table and added assertions: https://github.com/uwblueprint/social-diversity-for-children/pull/18/commits/dda4873e74143c478f092140a32161d2b7274d10